### PR TITLE
[cli] Unify table formatting output

### PR DESCRIPTION
**Pull request by onsclom**
This removes 'text-table' as a dependency in favor of using 'cli-table3', and simplifies table formatting logic.

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -83,7 +83,6 @@
     "@types/qs": "6.9.7",
     "@types/semver": "6.0.1",
     "@types/tar-fs": "1.16.1",
-    "@types/text-table": "0.2.0",
```
Comment (TooTallNate) [inline packages/cli/package.json:86]: Should we also remove the `text-table` dependency down below?

**Review by TooTallNate — COMMENTED** [review]
If there is any visual difference with this change, can you post before/after screenshots for comparison?

File: packages/cli/package.json (hunk 1 above)
Comment (onsclom) [reply packages/cli/package.json:86]: Good catch! Fixed now, thanks.

**Comment by onsclom** [comment]
> If there is any visual difference with this change, can you post before/after screenshots for comparison?

No visual differences. All tables should look the same.

**Comment by EndangeredMassa** [comment]
> No visual differences. All tables should look the same.


@onsclom can you post some before/after screenshots to show reviewers? If we had snapshot style tests around this, we could rely on those to be sure.

File: packages/cli/src/commands/teams/list.ts [hunk 2]
```diff
@@ -76,14 +77,22 @@ export default async function list(client: Client): Promise<number> {
...
   );
+  client.stdout.write(
+    currentTeam
+      ? teamTable
+          .split('\n')
+          .map((line, i) => `${i > 0 ? teamList[i - 1].prefix : ' '} ${line}`)
+          .join('\n')
+      : teamTable
+  );
+  client.stdout.write('\n');
```
Comment (jeffsee55) [inline packages/cli/src/commands/teams/list.ts]:
```suggestion
  client.stderr.write('\n');
```
I _think_ we pretty much always want to write to `stderr`

**Comment by onsclom** [comment]
Here are some before/after pics that show the output does not change:

## Before
alias-ls-before
teams-ls-before

## After
alias-ls-after
teams-ls-after

`vc alias ls` is an example using the new unified table which all the other tables use. Most tables were a simple transition like `vc alias ls`. `vc teams ls` is a special case because that one was hand rolled and now uses that unified table.

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**